frameworks/helloworld/src/main/dist/examples/gpu_resource.yml

```diff
@@ -0,0 +1,57 @@
+name: hello-world
+scheduler:
+  gpu-optin: true
```

Comment:
This is not the right way to specify GPU opt-in behavior in the YAML.  GPU resources should be specified in resource-sets like every other resource-set.